test/lit/passes/code-folding-eh-legacy.wast

```diff
@@ -377,13 +382,15 @@
         (if
           (pop i32)
           (then
+            (nop)
             (drop
               (i32.eqz
                 (i32.const 1)
               )
             )
           )
           (else
+            (nop)
```

Comment:
Why is this test changed? It looks like it should have been optimized both before and after this PR, so I think I'm missing something.

Reply:
Since the arms were not previously blocks, they would no longer be optimized. Adding nops makes the arms blocks so they continue to be optimized.